doc/whats_new/v0.24.rst

```diff
@@ -25,7 +25,14 @@ random sampling procedures.
 - |Fix| :class:`decomposition.KernelPCA` behaviour is now more consistent
   between 32-bits and 64-bits data when the kernel has small positive
   eigenvalues.
-
+- |Fix| In ``fit`` and ``fit_transform`` methods of Spectral bi-clustering 
+  algorithms :class:`sklearn.cluster.SpectralCoclustering` and 
+  :class:`sklearn.cluster.SpectralBiclustering`, and dimensionality reduction 
```

Comment:
We do not need to mention `SpectralCoclustering` or `SpectralBiclustering` because they are not changed in this PR.